cypress/e2e/recieve-notice/recieve-notice-triggers.cy.js

```diff
@@ -0,0 +1,311 @@
+    ServicePoints.deleteViaApi(testData.userServicePoint.id);
+    cy.deleteLoanPolicy(loanPolicyId);
+    NoticePolicyApi.deleteApi(testData.ruleProps.n);
+    Users.deleteViaApi(userData.userId);
+    PatronGroups.deleteViaApi(patronGroup.id);
+    cy.get('@items').each((item, index) => {
+      cy.deleteItem(item.itemId);
+      cy.deleteHoldingRecordViaApi(itemsData.itemsWithSeparateInstance[index].holdingId);
+      InventoryInstance.deleteInstanceViaApi(itemsData.itemsWithSeparateInstance[index].instanceId);
+    });
+    Location.deleteViaApiIncludingInstitutionCampusLibrary(
+      testData.defaultLocation.institutionId,
+      testData.defaultLocation.campusId,
+      testData.defaultLocation.libraryId,
+      testData.defaultLocation.id
+    );
+    NoticePolicyTemplateApi.getViaApi({ query: `name=${checkOutTemplate.name}` }).then((templateId) => {
+      NoticePolicyTemplateApi.deleteViaApi(templateId);
+    });
+    NoticePolicyTemplateApi.getViaApi({ query: `name=${loanDueDateChangeTemplate.name}` }).then((templateId) => {
+      NoticePolicyTemplateApi.deleteViaApi(templateId);
+    });
+    NoticePolicyTemplateApi.getViaApi({ query: `name=${checkInTemplate.name}` }).then((templateId) => {
+      NoticePolicyTemplateApi.deleteViaApi(templateId);
+    });
+  });
+
+  it(
+    'C347862 Check out + Loan due date change + Check in triggers (vega)',
+    { tags: [TestTypes.smoke, devTeams.vega] },
+    () => {
+      const createPatronNoticeTemplate = (template) => {
+        NewNoticePolicyTemplate.startAdding();
+        NewNoticePolicyTemplate.checkInitialState();
+        NewNoticePolicyTemplate.addToken('item.title');
+        NewNoticePolicyTemplate.create(template, false);
+        NewNoticePolicyTemplate.checkPreview();
+        NewNoticePolicyTemplate.saveAndClose();
+        NewNoticePolicyTemplate.waitLoading();
+        NewNoticePolicyTemplate.checkAfterSaving(template);
+      };
+      createPatronNoticeTemplate(checkOutTemplate);
+      createPatronNoticeTemplate(loanDueDateChangeTemplate);
+      createPatronNoticeTemplate(checkInTemplate);
+
+      cy.visit(settingsMenu.circulationPatronNoticePoliciesPath);
+      NewNoticePolicy.waitLoading();
+      NewNoticePolicy.startAdding();
+      NewNoticePolicy.checkInitialState();
+      NewNoticePolicy.fillGeneralInformation(noticePolicy);
+      NewNoticePolicy.addNotice(noticePolicy.selectOptions(checkOutTemplate));
+      NewNoticePolicy.addNotice(noticePolicy.selectOptions(loanDueDateChangeTemplate), 1);
+      NewNoticePolicy.addNotice(noticePolicy.selectOptions(checkInTemplate), 2);
+      NewNoticePolicy.save();
+      NewNoticePolicy.waitLoading();
+      NewNoticePolicy.check(noticePolicy);
+
+      cy.getNoticePolicy({ query: `name=="${noticePolicy.name}"` }).then((res) => {
+        testData.ruleProps.n = res[0].id;
+        testData.ruleProps.l = loanPolicyId;
+        testData.ruleProps.l = CirculationRules.addRuleViaApi(
```

Comment:
Does method CirculationRules.addRuleViaApi return somethig? Do you need it to save the result of this method to the property testData.ruleProps.l?